src/main/java/space/obminyashka/items_exchange/repository/enums/EmailUrls.java

```diff
@@ -0,0 +1,41 @@
+package space.obminyashka.items_exchange.repository.enums;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonValue;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+
+import java.util.Arrays;
+
+import static space.obminyashka.items_exchange.rest.api.ApiKey.EMAIL_VALIDATE_CODE;
+import static space.obminyashka.items_exchange.rest.api.ApiKey.USER_SERVICE_PASSWORD_CONFIRM;
+import static space.obminyashka.items_exchange.rest.response.message.MessageSourceProxy.getMessageSource;
+import static space.obminyashka.items_exchange.rest.response.message.ResponseMessagesHandler.ValidationMessage.INVALID_ENUM_VALUE;
+
+@Slf4j
+@RequiredArgsConstructor
+public enum EmailUrls {
```

Comment:
You can simply add a new parameter to existing enum EmailType instead of creation a new one.

Reply:
added